### PR TITLE
Fix global services not starting on macOS Sonoma

### PR DESCRIPTION
**Pull request by kierenevans**
My gut feeling is that macOS Sonoma does not like files named `something.service`.

When calling `ws global service proxy restart` for example, the output is nothing, the exit code is 0, it just does nothing!

When experimenting, copying the file to a different name let it execute, however as the code referred to `ws.service` for launching services, we couldn't get things fully working.
It may also be the way the home directory folder gets installed, so this may not be the issue.

Let's test this out for people that are having this issue.

Also fixed an issue with #149 on my system where `variables_order=GPCS` instead of `variables_order=EGPCS` in the system php.ini.

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
what's strange though is it works fine for me on MacOS Sonoma, although aws it uses my host aws

dash is definitely more standard for cli plugin systems though

File: src/Installer/Installer.php [hunk 1]
```diff
@@ -22,6 +22,10 @@ public static function install(string $home)
         passthru('chmod +x ' . $path . '/lib/*.sh');
         passthru('chmod +x ' . $path . '/service/*/init.sh');
 
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-aws ./ws.aws', $path));
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-poweroff ./ws.poweroff', $path));
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-service ./ws.service', $path));
+
```
Comment (andytson-inviqa) [inline src/Installer/Installer.php:28]: what would we use the . version for after this?

Comment (kierenevans) [reply src/Installer/Installer.php:28]: Just in case something in the harnesses use it. I don't think we do though, only `ws.aws`.

**Review by andytson-inviqa — DISMISSED** [review]

File: src/Interpreter/Executors/Bash/Executor.php [hunk 2]
```diff
@@ -95,4 +97,12 @@ private function buildCommand(string $script, array $args, ?string $cwd): array
             ...array_values($args),
         ];
     }
+
+    /**
+     * @return array<string,string>
+     */
+    private function buildEnv(): array
+    {
+        return getenv();
```
Comment (andytson-inviqa) [inline src/Interpreter/Executors/Bash/Executor.php:106]: it is apparently, since 8.0, but try
```suggestion
        return getenv(null, true);
```

Comment (andytson-inviqa) [reply src/Interpreter/Executors/Bash/Executor.php:106]: maybe it's default null is just not supplied and confused

Comment (kierenevans) [reply src/Interpreter/Executors/Bash/Executor.php:106]: I tried that locally but phpstan wasn't having it (even latest phpstan version, even with the phpVersion config in phpstan.neon).

Comment (andytson-inviqa) [reply src/Interpreter/Executors/Bash/Executor.php:106]: sorry joke incoming
```suggestion
        $env = [];
        foreach (array_keys(getenv()) as $key) {
            $value = getenv($key, true);
            if ($value !== false) {
                 $env[$key] = $value;
            }
        }
        return $env;
```

at least no http request variables in cli though so nvm

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**